src/analytics/src/main/scala/osmesa/analytics/stats/ChangesetStatsForeachWriter.scala

```diff
@@ -41,7 +41,7 @@ class ChangesetStatsForeachWriter(databaseUri: URI,
       |  measurements = (
       |    SELECT jsonb_object_agg(key, value)
       |    FROM (
-      |      SELECT key, sum(value::numeric) AS value
+      |      SELECT key, sum((value->>0)::numeric) AS value
```

Comment:
This was aggravating to figure out.  Shame SQL doesn't have static type analysis.

Reply:
Did this not previously work? Values should be scalar.

My objection to `(value->>0)::numeric` is that it casts a JSON number to a string (`->>0` vs. `->0`) and then back to a number vs. just casting the JSON number to a numeric.

```
seth=# select pg_typeof(('[12.4]'::jsonb)->0);
 pg_typeof
═══════════
 jsonb
(1 row)
seth=# select pg_typeof(('[12.4]'::jsonb)->>0);
 pg_typeof
═══════════
 text
(1 row)
```

Reply:
Can you dump `measurements` from one of the rows? I want to understand how values became arrays.

Reply:
N/m. I was developing against pg 11; prod is 10.6.

10.6:

```
=> select key, sum(value::numeric) from jsonb_each((select counts from changesets where id = <something>)) group by key;
ERROR:  cannot cast type jsonb to numeric
LINE 1: select key, sum(value::numeric) from jsonb_each((select coun...
```

11.4:

```
=# select key, sum(value::numeric) from jsonb_each((select counts from changesets where id = 3029981)) group by key;
      key       │ sum
════════════════╪═════
 roads_added    │   2
 roads_modified │  18
(2 rows)
```